test/parallel/test-http2-compat-serverresponse-end.js

```diff
@@ -19,6 +19,9 @@ const {
   // but may be invoked repeatedly without throwing errors.
   const server = createServer(mustCall((request, response) => {
     strictEqual(response.closed, false);
+    response.on('finish', mustCall(() => process.nextTick(
+      mustCall(() => doesNotThrow(() => response.end('test', mustNotCall())))
```

Comment:
Nit (feel free to ignore): mustCall seems redundant here. `process.nextTick()` callback is always called.